src/components/Exercise/ButtonEdit.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
-import Button from '../common/Button'
+import { Button } from '../common/Button'
 import { Link } from 'react-router-dom'
 
 export default function ButtonEdit() {
   return (
-    <Button mla="true" as={Link} to={`/exercises/edit`}>
+    <Button primary="true" mla="true" as={Link} to={`/exercises/edit`}>
```

Comment:
Maybe there is a way to get rid of the `mla` prop? I think for boolean props the presence of the key is enough.

Reply:
primary and mla are both styled components props, primary defines color and mla is margin left auto helper, you just have to write it like primary="true" because it's a LINK DOM element

Reply:
My thinking was, that primary carries some semantic meaning, while mla is just a styling helper.